1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/password_validate.ts

```diff
@@ -0,0 +1,27 @@
+interface IPasswordResult {
+  isValid: boolean,
+  errors: string[];
+}
+
+export function validatePassword(password: string): IPasswordResult {
+  let passwordResult: IPasswordResult = { isValid: false, errors: [] };
```

Comment:
```suggestion
  let passwordResult: IPasswordResult = { isValid: true, errors: [] };
```
I would set isValid = true as initial state just for consistency because errors array empty with isValid as false is not a valid state.

Reply:
agree, thanks!